ApiDocs.DocumentationGeneration/Templates/resource.md.mustache

```diff
@@ -0,0 +1,29 @@
+﻿# {{Name}} resource type
```

Comment:
Is there a template for methods/functions/collections/get/post/delete, etc, or is that generation not included yet?